openrtx/src/protocols/M17/M17FrameDecoder.cpp

```diff
@@ -72,93 +75,95 @@ M17FrameType M17FrameDecoder::decodeFrame(const frame_t& frame)
     return type;
 }
 
-M17FrameType M17FrameDecoder::getFrameType(const std::array< uint8_t, 2 >& syncWord)
+M17FrameType
+M17FrameDecoder::getFrameType(const std::array<uint8_t, 2> &syncWord)
 {
     // Preamble
-    M17FrameType type   = M17FrameType::PREAMBLE;
-    uint8_t minDistance = hammingDistance(syncWord[0], 0x77)
-                        + hammingDistance(syncWord[1], 0x77);
+    M17FrameType type = M17FrameType::PREAMBLE;
+    uint8_t minDistance =
```

Comment:
I would _really_ like to know why that damn thing of clang-format is doing these disgraces. It is also interesting that it changed this line, but the others remained untouched.
If you have the patience to try understanding how to make the tool formatting the lines like it was before, it'd be super. Otherwise, we can do a clang-format pass, fix these flaws by hand and do not add the C++ files to the list of automatic checks.